org.eclipse.lsp4j/src/main/java/org/eclipse/lsp4j/Protocol.xtend

```diff
@@ -816,13 +816,20 @@ class PublishDiagnosticsCapabilities {
 	 * Whether the client accepts diagnostics with related information.
 	 */
 	Boolean relatedInformation
+	
+	Boolean tagSupport
```

Comment:
Please add the comment as written in the vscode source:
```
	/**
	 * Client supports the tag property to provide meta data about a diagnostic.
	 *
	 * Since 3.15
	 */
```

Reply:
Added.